src/Components/HeaderComponents/HeaderComponents.tsx

```diff
@@ -0,0 +1,56 @@
+import { router } from 'expo-router';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import BackArrow from '../../../assets/red-left-caret.svg';
+import ShareButton from '../../../assets/red-share-button.svg';
+
+export const CasesHeader = () => {
+  return (
+    <Image
+      source={require('../../../assets/inline-logo.jpeg')}
+      style={{ width: 120, height: 15 }}
+    />
+  );
+};
+
+export const AllCasesHeader = () => {
+  return (
+    <View>
+      <View style={styles.allCasesBorder}>
+        <CasesHeader />
+      </View>
+    </View>
+  );
+};
+interface backButtonProps {
+  backText: string;
+}
+export const BackButton = ({ backText }: backButtonProps) => {
```

Comment:
If there's only a few props, it might be easier to directly declare the function arguments like `(backText: string)`.